package.json

```diff
@@ -21,6 +22,7 @@
     "fence": "guardian/fence#0.2.11",
     "lodash-amd": "~2.4.1",
     "ophan-tracker-js": "1.3.9",
+    "postcss-css-variables": "^0.8.0",
```

Comment:
if the idea of atoms is that they should be autonomous and untouched by changes to front end, should this be a dependency of atom renderer?

Reply:
Would it still work? This is unique to the atom lib + frontend combo, apps do it differently. Atoms are not autonomous, they rely on a platform to provide essential features; what I did was abstract these features away.